docs/app/pages/css/css-sections/icons/ux-icons/ux-icons.component.html

```diff
@@ -1,28 +1,79 @@
 <p *ngIf="iconSetClass === 'hpe-icon'">To use an icon simply apply the <code>{{ iconSetClass }}</code>
     class along with one of the following classes:</p>
 
-<p  *ngIf="iconSetClass === 'ux-icon'">To use an icon apply the <a fragment="icon"
+<p *ngIf="iconSetClass === 'ux-icon'">To use an icon apply the <a fragment="icon"
     [routerLink]="[uxIconComponentRoute]"><code>ux-icon</code> component</a> along with one of the
     following names. Click on an icon to view its implementation.</p>
 
 <div class="row m-b-md">
-    <div class="col-xs-6">
 
-        <!-- Show the standard select component in the keppel site -->
-        <ux-select *ngIf="isKeppel" [(value)]="iconset" (valueChange)="updateIconset(iconset); search()" [options]="iconsets" [readonlyInput]="true"></ux-select>
+    <div class="col-lg-4 col-md-12 col-sm-12">
+
+        <div class="icon-search-items-container">
+
+            <input #input
+                   type="text"
+                   class="form-control"
+                   placeholder="Search UX Aspects Icons..."
+                   [(ngModel)]="query"
+                   (ngModelChange)="search()">
+
+        </div>
+    </div>
+
+    <div class="col-lg-4 col-md-12 col-sm-12">
```

Comment:
Just noticed one issue with text wrapping when the hpe-icon set is selected:

![image](https://user-images.githubusercontent.com/20795331/63092849-0480f480-bf5b-11e9-8543-496e4e01c479.png)

Reply:
fixed